alns/accept/tests/test_worse_accept.py

```diff
@@ -1,22 +1,11 @@
-import numpy as np
+from unittest.mock import Mock
+
 import numpy.random as rnd
 from numpy.testing import assert_, assert_equal, assert_raises
 from pytest import mark
 
 from alns.accept import WorseAccept
-from alns.tests.states import Zero, One, Two
-
-
-class MockRNG:
-    """
-    Mock Random Number Generator.
-    """
-
-    def __init__(self, values):
-        self.values = values
-
-    def random(self):
-        return self.values.pop(0)
```

Comment:
@leonlan I've replaced this with [`unittest.Mock`](https://docs.python.org/3/library/unittest.mock.html), which is a great built-in class for these sorts of things. I wanted to try Mock out for some time, and figured this would be a good place to start. It works really well!